doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,9 +41,16 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml>  Specifies whether to generate HTML or XML output.
+  -f, --format <html|xml|xml:html>
+                           Specifies whether to generate HTML or XML output.
                            If this options is not used, the format is got
                            from the extension of the output file.
+                           Embedding HTML documentation in XML output:
+                           If XML should have embedded HTML in <doc> as
+                           documentation, it is possible to configure
+                           this with ``XML:HTML``. This could be used if the XML
+                           files are used to be imported into other systems
+                           that require HTML text to present this to a human.
```

Comment:
I don't think it's adequate to explain this kind of a feature only in the command line usage. Features should be documented so that the docs can be found easily.

The problem is that Libdoc documentation doesn't seem to have a good place to explain this. Probably the best place would be under the existing `Generating documentation` section, but that section should also be enhanced to explain HTML and XML formats a bit more thoroughly. I can do that if you want.

There should also be a note that this is a new feature in RF 3.2.

Reply:
I would really appreciate, if you could add this to Libdoc documentation.
I wanted to do this but as you mentioned i could not fit this into the existing structure.
I was hard for me to "restructure" your docs.

I would recomment only to add the "new in RF 3.2" comment only in the libdoc CLI doc.
I think it really overloads the manpage without adding value to the user who is already using 3.2 when he reads this information.

Reply:
added "``XML:HTML`` is new in RF 3.2." into libdoc.rst